corehq/apps/reports/v2/endpoints/case_properties.py

```diff
@@ -37,16 +53,66 @@ def case_properties(self):
         ]
         return case_properties + special_properties
 
+    @staticmethod
+    def _fmt_select2(prop):
+        prop['text'] = prop['name']
+        prop['id'] = prop['name']
+        prop['labelStyle'] = ('label-default' if prop['case_type']
+                              else 'label-primary')
+        prop['labelText'] = prop['case_type'] or prop['meta_type']
```

Comment:
You probably saw this in the current CLE, but we also return the count of the number of case types with the same case property name and change this label to match, otherwise it could be a bit misleading

https://github.com/dimagi/commcare-hq/blob/aecd6698d482eb9eeda62d59565f26c35931f590/corehq/apps/reports/static/reports/js/filters/case_list_explorer.js#L22

It was hard to see from the small screenshot how this shows up currently. 

eg:
![Screenshot from 2019-05-13 14-17-16](https://user-images.githubusercontent.com/146896/57644314-de2d7080-7589-11e9-9df9-e7f3849f6b45.png)